src/Tests/AddressTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+
+namespace GingerPluginSdk\Tests;
+
+use GingerPluginSdk\Entities\Address;
+use GingerPluginSdk\Properties\Country;
+use GingerPluginSdk\Exceptions\OutOfEnumException;
+use GingerPluginSdk\Exceptions\OutOfPatternException;
+use GingerPluginSdk\Helpers\SingleFieldTrait;
+use PHPUnit\Framework\TestCase;
+use TypeError;
+
+class AddressTest extends TestCase
+{
+    private Address $address;
+
+    public function __construct(?string $name = null, array $data = [], $dataName = '')
```

Comment:
Setting up a test is supposed to be done in the setUp() method. This will be triggered on every individual test.

Reply:
Oh, idk that, thanks a lot it's what i want to do!